monitor/reducers/estateReducer.js

```diff
@@ -92,17 +92,17 @@ export async function estateReducer(events, event) {
       break
     }
     case events.estateTransfer: {
-      const { to, estateId } = event.args
+      const { _to, _tokenId } = event.args
 
-      log.info(`[${name}] Transfering "${estateId}" owner to "${to}"`)
+      log.info(`[${name}] Transfering "${_tokenId}" owner to "${_to}"`)
 
       const last_transferred_at = await new BlockTimestampService().getBlockTime(
         block_number
       )
 
       await Estate.update(
-        { owner: to.toLowerCase(), last_transferred_at },
-        { asset_id: estateId }
+        { owner: _to.toLowerCase(), last_transferred_at },
+        { asset_id: _tokenId }
```

Comment:
Do we have a companion PR on https://github.com/decentraland/land for this?

Reply:
No, `EstateRegistry` inherit from the [ERC721 standard](https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/token/ERC721/ERC721Basic.sol#L42) which `transfer` method has that names for `args`